### DISCUSSION TITLE
Change the name of tab

### DISCUSSION THREAD

**Discussion by dega5** [zauberzeug/nicegui#1939] [Q&A] [answered]
### Question

Hi! 

Does anyone know how to change the name of the tabg in web browser from nicegui to something custom?

![09_53_43](https://github.com/zauberzeug/nicegui/assets/96119834/9f1e1bda-60a5-469c-8ab0-a22c845575ee)

Thank you!!

**Comment by phoskee** [comment, marked as answer]
```py
from nicegui import ui

ui.label('page with custom title')

ui.run(title='My App')
```



[Page](https://nicegui.io/documentation#page)

**Comment by ed2050** [comment]
More generally, you can set titles on different pages with `ui.page`.  Using `ui.run` only sets 1 title across the entire app.

```py
@ui.page ('/foo', title = 'My Page')
def mypage () :
    # do something

@ui.page ('bar', title = 'Another Page')
def anotherpage () :
    # do something
```

There is no way at present to set or change a page title dynamically with nicegui.  Unfortunately that [requires javascript](https://github.com/zauberzeug/nicegui/discussions/1436#discussioncomment-6746685).
